packages/slate-editor/src/components/EditorBlock/Menu.tsx

```diff
@@ -108,37 +112,39 @@ export class Menu extends Component<Props> {
 
     render() {
         const { children, className, onClick, popperOptions } = this.props;
-        const placement = popperOptions.placement || 'right-start';
 
         return (
             <Popper
                 referenceElement={{
                     getBoundingClientRect: this.getVirtualReferenceClientRect,
                 }}
                 modifiers={getModifiers(popperOptions)}
-                placement={placement}
+                placement="right-start"
                 strategy="fixed"
             >
-                {({ ref, style, arrowProps, placement }) => (
-                    <Toolbox.Panel
-                        contentEditable={false}
-                        className={classNames(className, styles.menu)}
-                        ref={ref}
-                        style={style}
-                        onClick={onClick}
-                    >
-                        <div
-                            className={classNames(styles.arrow, {
-                                [styles.top]: placement.indexOf('top') >= 0,
-                                [styles.bottom]: placement.indexOf('bottom') >= 0,
-                                [styles.left]: placement.indexOf('left') >= 0,
-                                [styles.right]: placement.indexOf('right') >= 0,
-                            })}
-                            {...arrowProps}
-                        />
-                        {children}
-                    </Toolbox.Panel>
-                )}
+                {({ ref, style, arrowProps, placement }) =>
+                    createPortal(
+                        <Toolbox.Panel
+                            contentEditable={false}
+                            className={classNames(className, styles.menu)}
+                            ref={ref}
+                            style={style}
+                            onClick={onClick}
+                        >
+                            <div
+                                className={classNames(styles.arrow, {
+                                    [styles.top]: placement.indexOf('top') >= 0,
+                                    [styles.bottom]: placement.indexOf('bottom') >= 0,
+                                    [styles.left]: placement.indexOf('left') >= 0,
+                                    [styles.right]: placement.indexOf('right') >= 0,
+                                })}
+                                {...arrowProps}
+                            />
+                            {children}
+                        </Toolbox.Panel>,
+                        document.getElementById('full-page-react-application') as HTMLElement,
```

Comment:
There should be some container from props

Reply:
I think we could add the prop into the context I've introduced.

Reply:
I was able to make it work just by providing `document.body` here. Maybe we can use that for now and add the prop in the future when/if needed?

Reply:
Yeah, let's leave for now. The one problem could be if user's browser extensions will mutate body's elements, but it is an edge case for now